commons-okhttp/src/test/scala/com/avsystem/commons/okhttp/rest/RestTest.scala

```diff
@@ -0,0 +1,65 @@
+package com.avsystem.commons
+package okhttp.rest
+
+import com.avsystem.commons.jetty.rest.RestServlet
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.servlet.{ServletHandler, ServletHolder}
+import org.scalatest.FunSuite
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.{Seconds, Span}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class RestTest extends FunSuite with UsesJettyHttpServer with UsesOkHttpClient with ScalaFutures {
```

Comment:
Can we merge this test with `AbstractRestCallTest`? I'd like to have a single test suite tested with different client/server implementations.